### PR TITLE
Fix get_started links and brew install one-liner

### PR DESCRIPTION
**Pull request by scpeters**

### CODE REVIEW

**Comment by scpeters** [comment]
I'm going to propagate these changes to other similar files

File: citadel/install_ubuntu.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ sudo apt-get install ignition-citadel
 
 All libraries should be ready to use and the `ign gazebo` app ready to be executed.
 
-Head back to the [Getting started](/docs/get_started)
+Head back to the [Getting started](/get_started)
```
Comment (chapulina) [inline citadel/install_ubuntu.md:15]: I think the correct link is `/docs/all/get_started`

File: dome/install_osx.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ brew install ignition-dome
 
 All libraries should be ready to use and the `ign gazebo` app ready to be executed.
 
-Head back to the [Getting started](/docs/get_started)
+Head back to the [Getting started](/get_started)
```
Comment (chapulina) [inline dome/install_osx.md:22]: I think the correct link is `/docs/all/get_started`

File: dome/install_ubuntu.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ sudo apt-get install ignition-dome
 
 All libraries should be ready to use and the `ign gazebo` app ready to be executed.
 
-Head back to the [Getting started](/docs/get_started)
+Head back to the [Getting started](/get_started)
```
Comment (chapulina) [inline dome/install_ubuntu.md:18]: I think the correct link is `/docs/all/get_started`

**Review by chapulina — CHANGES_REQUESTED** [review]

File: citadel/install_ubuntu.md (hunk 1 above)
Comment (scpeters) [reply citadel/install_ubuntu.md:15]: thanks, I was looking at the GitHub folder structure

I think the links won't work from GitHub though

File: dome/install_ubuntu.md (hunk 3 above)
Comment (scpeters) [reply dome/install_ubuntu.md]: 5a19708659f5f91470480ad9f8dfc9238c2ba85c

File: dome/install_osx.md (hunk 2 above)
Comment (scpeters) [reply dome/install_osx.md]: 5a19708659f5f91470480ad9f8dfc9238c2ba85c

File: citadel/install_ubuntu.md (hunk 1 above)
Comment (scpeters) [reply citadel/install_ubuntu.md]: 5a19708659f5f91470480ad9f8dfc9238c2ba85c

Comment (chapulina) [reply citadel/install_ubuntu.md]: Ticketed a related issue: https://github.com/ignitionrobotics/docs/issues/85

**Review by chapulina — APPROVED** [review]

**Pull request merged**